crossbeam-epoch/src/collector.rs

```diff
@@ -109,7 +109,7 @@ impl fmt::Debug for LocalHandle {
     }
 }
 
-#[cfg(test)]
+#[cfg(all(test, not(loom)))]
```

Comment:
Tests are excluded from loom.  Is it intended?

Reply:
Yes indeed. When we're running under loom, we _only_ want to run the loom tests that have the loom "runtime" (in the form of `loom::model`).